### ISSUE TITLE
Length of numeric fields

### ISSUE THREAD

**Issue by maurizio737** [DotSpatial/DotSpatial#1220]
I need in c# when I create a shapefile to define the length of the characters in integer field, if I use (int32) in the code I am created in the .dbf a length equal to 11, here is my code:
FeatureSet fsSet = new FeatureSet(FeatureType.Line);
fsSet.CoordinateType = CoordinateType.Z;
Feature f = new Feature();
System.Data.DataColumn DIAMETER = new System.Data.DataColumn("DIAMETER", typeof(Int32));
fsSet.DataTable.Columns.Add(DIAMETER);
DotSpatial.Topology.Point point = new DotSpatial.Topology.Point(1, 1, 0);
Feature f = new Feature(point);
fsSet.Features.Add(f);
f.DataRow.BeginEdit();
f.DataRow["DIAMETER"] = 32;
f.DataRow.EndEdit();
fsSet.Projection = KnownCoordinateSystems.Projected.UtmWgs1984.WGS1984UTMZone33N;
fsSet.SaveAs(lbPercorso.Text + "\\XXXXX.shp", true);

**Comment by sindizzy** [comment]
What is the purpose of the length? Is it for displaying it on the map?

**Comment by maurizio737** [comment]
No, I have to import it into a software that also checks the length of the integer fields

**Comment by maurizio737** [comment]
I currently use third-party software to change this length

**Comment by sindizzy** [comment]
Use the Field object to better define your field https://github.com/DotSpatial/DotSpatial/issues/221
